core/src/main/scala/scabot/core/Configuration.scala

```diff
@@ -63,9 +63,10 @@ trait Configuration extends Core {
         repo  <- configString(c.get("repo"))
         branches  <- configStringList(c.get("branches"))
         lastCommitOnly <- configBoolean(c.get("lastCommitOnly")) orElse Some(false) // default for scala/scala
+        checkCLA <- configBoolean(c.get("checkCLA")) orElse Some(true) // default for scala/scala
```

Comment:
shouldn't the default be `false`, for backwards compat? We'll also need to update scala-jenkins-infra.